src/resources/js/checkout.js

```diff
@@ -0,0 +1,82 @@
+class Checkout {
```

Comment:
@AlexeiaW great that these are pulled out into their own classes. This could make it easier to unit test. 

- For unit testing we can use jest
- Unit tests should not rely on the laravel environment. The problem with this is when we tightly couple our tests to an API or some third party, we're going to end up with failing tests if not all services are present. 
- Test as much as possible, and refactor your code so that you can test certain functionality that might be difficult to test without the API present or Laravel views. 
- You may need to import jquery to get testing working (not sure about external CDN includes).